### DISCUSSION TITLE
Camera support

### DISCUSSION THREAD

**Discussion by Undercore90** [alienatedsec/yi-hack-v5#361] [Q&A]
Hi,

I hope you can help me to customize your software for my camera. FTP and Telnet access works, but no method firmware on SD does not work.
[CAMERA.zip](https://github.com/alienatedsec/yi-hack-v5/files/14397776/CAMERA.zip)

**Comment by alienatedsec** [comment]
@Undercore90 A long time ago, I was in a similar position and wanted to build firmware. It's not easy and not straightforward. 

First, you need an SDK for your camera’s chipset. Second, you need to recompile the software. Third you need to tweak and integrate with your camera-specific settings and model. 

I am not sure how far you have gone, but it's unlikely I can help with a positive outcome in the short term.

**Comment by Undercore90** [comment]
https://github.com/MuhammedKalkan/Anyka-Camera-Firmware

This version runs from the card. gc1084 is the module that is in the camera. I am interested in ONVIF so that the Synology can record and also control the PTZ.

**Comment by premiero** [comment]
@alienatedsec - should Yi Home 1080p with 
![obraz](https://github.com/alienatedsec/yi-hack-v5/assets/27424909/d27c88f2-9848-4999-a57a-c9912f0beb36)
 code work with yi-hack-v5 or is it not supported? I tried to flash it with y20 version, but with no success (did not even start to flash).

**Reply by alienatedsec** [reply]
@premiero RFUS is the Allwinner camera. Check @roleoroleo repos

**Reply by premiero** [reply]
Tx mate, y291ga_0.3.1 works perfectly.

**Comment by RICHIG81** [comment]
Hello!

I am trying to carry out the process with a Yi CAM DOME U 2K and I am not succeeding with the V5 version. Is it supported? Is there any previous step that I haven't found?
Thank you so much

**Reply by alienatedsec** [reply]
@RICHIG81 not sure which model is that. The first 4 chars of the serial number will be useful

**Reply by RICHIG81** [reply]
YI DOME CAMERA U
DID: BFUSY45MVM
FCC ID: 2AFIB-YHS5020
IC: 20436.YHS5020

This is what it says. Thanks for answering so quickly

**Reply by alienatedsec** [reply]
Try @roleoroleo repos. It looks like Allwinner camera.

**Reply by RICHIG81** [reply]
Thanks, I'll try it
